src/ansys/health/heart/models.py

```diff
@@ -794,24 +825,139 @@ def save_model(self, filename: str):
         extension = pathlib.Path(filename).suffix
         if extension != "":
             mesh_path = filename.replace(extension, ".vtu")
-            map_path = filename.replace(extension, ".partinfo.json")
+            info_path = filename.replace(extension, ".partinfo.json")
         else:
             mesh_path = filename + ".vtu"
-            map_path = filename + ".partinfo.json"
+            info_path = filename + ".partinfo.json"
 
         self.mesh.save(mesh_path)
 
-        with open(map_path, "w") as f:
+        with open(info_path, "w") as f:
             json.dump(self._get_parts_info(), f, indent=4)
 
-        return
+        return mesh_path, info_path
 
-    # TODO: could consider having this as a static method.
-    # TODO: Right now this only reconstructs the surfaces and parts that
-    # TODO: are defined in the HeartModel classes:
-    # TODO: LeftVentricle, BiVentricle, FourChamber and FullHeart.
-    # TODO: Should consider to also reconstruct the parts that are not explicitly
-    # TODO: defined in the class.
+    @staticmethod
+    def load_model(
+        filename_mesh: str,
+        filename_part_info: str,
+        working_directory: pathlib.Path | str = None,
+    ) -> FullHeart | FourChamber | BiVentricle | LeftVentricle:
+        """Load a heart model from an existing VTU file and part information dictionary.
+
+        Parameters
+        ----------
+        filename_mesh : str
+            Path to the VTU file containing the mesh.
+        filename_part_info : str
+            Path to the JSON file that contains the part information for reconstructing the model..
+        working_directory : pathlib.Path | str, default: None
+            Working directory.
+
+        Returns
+        -------
+        FullHeart | FourChamber | BiVentricle | LeftVentricle
+            A instance of HeartModel.
```

Comment:
```suggestion
            Instance of the heart model.
```
Don't start a brief description with an article (The, A, or An). Use common language in brief descriptions (rather than code entities) where possible.